code-examples/helm-kafka-example/README.md

```diff
@@ -33,10 +33,28 @@ $ oc apply -f ./rhoas-secrets.yaml
 $ oc apply -f ./rhoas-services.yaml
 ```
 
-5. Deploy the helm chart setting the appropriate values:
- 
-```
-$ helm install . --generate-name --set-string rhoas.config=<configmap-name>,rhoas.secret=<secret-name>
-```
+5. Deploy the helm chart with the appropriate values:
```

Comment:
It made more sense to include more details in the step itself rather than having a separate sub-heading for it. Let me know if you feel if it should be moved to a higher level @jbyrne-redhat

Reply:
Yes, that makes sense to me!